lib/src/main/java/com/futurewei/alcor/common/enumClass/RouteTableType.java

```diff
@@ -20,7 +20,8 @@
     PUBLIC_SUBNET("public_subnet"),
     PRIVATE_SUBNET("private_subnet"),
     VPC("vpc"),
-    NEUTRON("neutron");
+    NEUTRON_ROUTER("neutron_router"),
```

Comment:
@kevin-zhonghao @cj-chung We might want to describe each enum value a bit more, either in the code or in the design doc.

Reply:
Agree